components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/SftpConfiguration.java

```diff
@@ -269,4 +271,16 @@ public LoggingLevel getJschLoggingLevel() {
     public void setJschLoggingLevel(LoggingLevel jschLoggingLevel) {
         this.jschLoggingLevel = jschLoggingLevel;
     }
+    
+    /**
+     * Specifies how many requests may be outstanding at any one time. Increasing this value may
+     * slightly improve file transfer speed but will increase memory usage.
+     */
+    public void setBulkRequests(Integer bulkRequests) {
+		this.bulkRequests = bulkRequests;
+	}
+    
+    public Integer getBulkRequests() {
+		return bulkRequests;
+	}
```

Comment:
Make sure the code passes checkstyle, run

mvn compile -Psourcecheck

to validate